2011-05-12  Michael Hope  <michael.hope@linaro.org>

	gcc/
	Backport from mainline:

	2011-05-05  Michael Hope  <michael.hope@linaro.org>

	PR pch/45979
	* config/host-linux.c (TRY_EMPTY_VM_SPACE): Define for
	__ARM_EABI__ hosts.

=== modified file 'gcc/config/host-linux.c'
--- old/gcc/config/host-linux.c	2010-11-29 14:09:41 +0000
+++ new/gcc/config/host-linux.c	2011-05-06 20:19:30 +0000
@@ -84,6 +84,8 @@
 # define TRY_EMPTY_VM_SPACE	0x60000000
 #elif defined(__mc68000__)
 # define TRY_EMPTY_VM_SPACE	0x40000000
+#elif defined(__ARM_EABI__)
+# define TRY_EMPTY_VM_SPACE	0x60000000
 #else
 # define TRY_EMPTY_VM_SPACE	0
 #endif

